index.html

```diff
@@ -55,7 +55,7 @@ <h2>第２回Git勉強会</h2>
                     <article>
                         <h3>イベント日時・場所</h3>
                         <p>3月23日 19:00開始</p>
-                        <p>株式会社〇〇 イベントセミナー会場</p>
+                        <p>株式会社 AnyCompany イベントセミナー会場</p>
```

Comment:
Japanese company name if possible?

Reply:
Looks no Japanese company name is available for this company. Please ignore above comment.